leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/RootResource.java

```diff
@@ -28,10 +29,11 @@
     public RootResource() {
         super("");
         setVisible(false);
+        setObservable(true);
```

Comment:
I'm not sure but I guess this is not intended this the client root resource which should be observable not the server one ?

Reply:
That's right, my mistake :)